frappe/public/js/frappe/widgets/new_widget.js

```diff
@@ -19,7 +19,8 @@ export default class NewWidget {
 	get_title() {
 		// DO NOT REMOVE: Comment to load translation
 		// __("New Chart") __("New Shortcut") __("New Number Card")
-		return __(`New ${frappe.model.unscrub(this.type)}`);
+		let title = `New ${frappe.model.unscrub(this.type)}`;
```

Comment:
will this be translatable?

Reply:
https://github.com/frappe/frappe/pull/11689#discussion_r503201704